website/docs/user-guides/chaoscenter-oauth-dex-installation.md

```diff
@@ -103,30 +103,32 @@ litmusportal-server-9c4d85f57-5r6km       2/2     Running             0
 mongo-0                                   1/1     Running             0          5m57s
 ```
 
-
 ### Configuring `litmusportal-server` to enable Dex features
 
 To set up Dex, we would require to modify our litmusportal-server a bit in order to communicate with Dex. This will be achieved by adding some environment variables
 
 - `OIDC_ISSUER`: The place where the Dex OIDC lives, i.e `NODE_IP:32000`
 - `DEX_ENABLED`: This variable enables dex features in the litmusportal-server
-- `CALLBACK_URL`: This is the url that will be called back after user completes thier OAuth, this will be the litmusportal-frontend service
+- `DEX_OAUTH_CALLBACK_URL`: This is the url that will be called back after user completes its authentication, this will be the litmusportal-frontend service
+- `DEX_OAUTH_CLIENT_ID`: The client ID defined at the DEX server to allow it act on behalf of Litmus, i.e: `LitmusPortalAuthBackend`
+- `DEX_OAUTH_CLIENT_SECRET`: The client secret defined at the DEX server, i.e: `ZXhhbXBsZS1hcHAtc2VjcmV0`
 
-Set your variables using 
+Set your variables using
 
 ```bash
-kubectl set env deployment/litmusportal-server -n litmus --containers="auth-server" DEX_SERVER="true", OIDC_ISSUER=<REPLACE_NODE_IP>:32000, CALLBACK_URL=<REPLACE_litmusportal_frontend_Service>
+kubectl set env deployment/litmusportal-server -n litmus --containers="auth-server" DEX_SERVER="true", OIDC_ISSUER=<REPLACE_NODE_IP>:32000, DEX_OAUTH_CALLBACK_URL=<REPLACE_litmusportal_frontend_Service>/callback,
+DEX_OAUTH_CLIENT_ID="LitmusPortalAuthBackend",
+DEX_OAUTH_CLIENT_SECRET="ZXhhbXBsZS1hcHAtc2VjcmV0"
```

Comment:
I think these credentials were intended to be configured [here](https://github.com/litmuschaos/litmus/blob/ff180aecf5a445c18d56ec10dfcd084c58f9fd52/litmus-portal/cluster-k8s-manifest.yml#L421-L426) during deployment, because if you change the env parameters of `DEX_OAUTH_CALLBACK_URL`, `DEX_OAUTH_CLIENT_ID` and `DEX_OAUTH_CLIENT_SECRET` on the fly, this will cause the whole deployment to restart (including GraphQL server, because its dependent on this configmap). That's the reason it wasn't added here initially.  

We can proceed with either of these approaches:
- change it on the fly and let the graphQL server restart too and warn the user about the same in the docs
- Just link the yaml file and give a headsup in the documentation to user to manually configure it in `cluster-k8s.yaml` file.

cc: @ksatchit @rajdas98

Reply:
Indeed! at the moment the documentation does not mention these attributes, being difficult to figure out that these attributes were needed.

Reply:
BTW in the docs the first option is already there:

Reply:
Yes, we might have to highlight it as an Warning before the operation instead

Reply:
Just highlighted that in the doc